tests/udp/test_udp_header_with_fixed_length_fixed_checksum.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+import utils
+from abstract_open_traffic_generator import flow as Flow
+
+
+@pytest.mark.parametrize('packets', [1000])
+@pytest.mark.parametrize('size', [74])
+def test_udp_header_with_fixed_length_fixed_checksum(api, b2b_raw_config, size, packets):
+    """
+    Configure a raw udp flow with,
+    - fixed src and dst Port address, length, checksum
+    - 1000 frames of 74B size each
+    - 10% line rate
+
+    Validate,
+    - tx/rx frame count is as expected
+    - all captured frames have expected src and dst Port address
+    """
+    flow = b2b_raw_config.flows[0]
+
+    flow.packet = [
+        Flow.Header(
+            Flow.Ethernet(
+                src=Flow.Pattern('00:0c:29:1d:10:67'),
+                dst=Flow.Pattern('00:0c:29:1d:10:71')
+                )
+            ),
+        Flow.Header(
+            Flow.Ipv4(
+                src=Flow.Pattern("10.10.10.1"),
+                dst=Flow.Pattern("10.10.10.2"),
+                )
+            ),
+        Flow.Header(
+            Flow.Udp(
+                src_port=Flow.Pattern("3000"),
+                dst_port=Flow.Pattern("4000"),
+                length=Flow.Pattern("38"),
+                checksum=Flow.Pattern("5")
+            )
+        ),
+    ]
+    flow.duration = Flow.Duration(Flow.FixedPackets(packets=packets))
+    flow.size = Flow.Size(size)
+    flow.rate = Flow.Rate(value=10, unit='line')
+
+    utils.start_traffic(api, b2b_raw_config)
+    utils.wait_for(
+        lambda: results_ok(api,size,packets), 'stats to be as expected'
+    )
+
+    captures_ok(api, b2b_raw_config, size)
+
+
+def results_ok(api, size, packets):
```

Comment:
stats_ok and results_ok in all the tests are same, it is better to move it to utils.py